tests/Basics/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 target_sources(arangodbtests
   PRIVATE
+  BuildIdTest.cpp
```

Comment:
I guess this test file is currently missing from the PR, so building the tests will likely fail.